src/components/team/Team.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import {useState} from 'react'
+import Button from "../button/Button"
```

Comment:
This should read `import Button from "../button"` because the button component would have had the index.js and the css.module.scss you do not need to name the javascript file Button

Reply:
Oh so leave the js files in each component folder as index.js then?